### ISSUE TITLE
Texture editing

### ISSUE THREAD

**Issue by lemonyPRIDE** [efroemling/ballistica#62]
I know it's still VERY MUCH EARLY but ..hey Eric, is it possible that offline editing of the textures can be edited for online servers too so that it makes the gameplay or whoever hosting the server UNIQUE?
I've been playing this game for almost 5yrs now and I personally feel it really is lacking some MORE colors😃.... just please make the editing available for 1.6update as I dont want you to delay it further😁

Imagine each server having their own unique Design, but playing on new freshly designed textured server. This will really bring a whole new level of attention for BOMBSQUAD COMMUNITY🤯

**Comment by efroemling** [comment]
Yes this exact feature is coming in [1.7](https://github.com/efroemling/ballistica/wiki/Roadmap#ballistica-17). It is already mostly done and was going to be my 'killer' feature for 1.5, but at some point I decided that I've delayed 1.5 long enough already so I'm pushing it out just a bit. (I'm doing 1.6 first to replace Google Play multiplayer connections since a lot of people are missing those)

I'm quite looking forward to seeing what people come up with once they no longer have to attempt to get people to install a bunch of modded asset files manually... And it'll also let me start adding more stuff too; I had been somewhat holding off just to keep the APK sizes small.

**Comment by Loup-Garou911XD** [comment]
Closing as planned for 1.8 asset packs
